tesseract.yaml

```diff
@@ -141,13 +134,16 @@ subpackages:
   - name: tesseract-dev
     pipeline:
       - uses: split/dev
+    dependencies:
+      runtime:
+        - tessdata
```

Comment:
For the header files we may not need tessdatas. look into [Old PKGINFO ](https://apk.dag.dev/https/packages.wolfi.dev/os/aarch64/tesseract-dev-5.4.1-r1.apk@sha1:d748216be999c43f8f5a57aa2d00bc017ac00cfc/.PKGINFO)
```suggestion
```

Reply:
Looks to be the case, committing and building.